pypeit/core/arc.py

```diff
@@ -998,6 +999,9 @@ def detect_lines(censpec, sigdetect=5.0, fwhm=4.0, fit_frac_fwhm=1.25, input_thr
     if input_thresh is None:
         (mean, med, stddev) = stats.sigma_clipped_stats(arc[cont_mask & np.logical_not(bpm_out)], sigma_lower=3.0, sigma_upper=3.0)
         thresh = med + sigdetect*stddev
+        if stddev == 0.0:
```

Comment:
It is sometimes the case that the sigma_clipped_stats function reports a 0.0 valued stddev, particularly if there are relatively few unmasked continuum points. This will cause the function to crash down the line when computing nsig = (tampl - med)/stddev. The better thing to do to preserve the intended function is to just reset it to 1.0.